conftest.py

```diff
@@ -181,7 +181,7 @@ def pytest_configure(config):
     it to reduce the verbosity of the test names in the table.  This leaves
     enough room to see the information about the test failure in the summary.
     """
-    global CONFIG
+    global CONFIG  # noqa: PLW0602
```

Comment:
Can we blanket disable this rule against globals?

Reply:
Doing it this way triggers the "Do we really need a global here?" discussion in code reviews and when refactoring but I will make the requested change.

Reply:
I think `global` by itself triggers a "Do we really need a global here?" discussion in review though. It is at least generally a short line.

Reply:
> Now that I see that this rule is violated only once in the entire codebase, I am even more reluctant to change but... here we go.

Yeah maybe you can put it back on seeing as it only happens once.

Reply:
Sorry.